pkgs/by-name/ms/msedgedriver/package.nix

```diff
@@ -0,0 +1,51 @@
+{
+  autoPatchelfHook,
+  fetchurl,
+  glib,
+  lib,
+  nspr,
+  nss,
+  stdenv,
+  unzip,
+  xorg,
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "msedgedriver";
+  version = "129.0.2792.91";
+
+  src = fetchurl {
+    url = "https://msedgedriver.azureedge.net/${finalAttrs.version}/edgedriver_linux64.zip";
+    hash = "sha256-69+quFNgjioVBtsRSZDPBYXWhtCvX6cZ5TqGhhf5Ht8=";
+  };
+
+  buildInputs = [
+    stdenv.cc.cc.lib
```

Comment:
It looks unnecessary, any specific reason why it was added?
```suggestion
```

Reply:
I copied the dependecies of the chromium driver since the two are basically equivalent

Reply:
lets see if it works without it

Reply:
it is still starting without it and seems to have all dependencies, ill remove it